src/main/java/recipeio/ui/UI.java

```diff
@@ -7,7 +7,8 @@
 import java.util.Scanner;
 
 public class UI {
-    public static final String SEPARATOR = "-----------------------------";
+    public static final String SEPARATOR = "-------------------------------------------------------------------------" +
+            "--------------";
```

Comment:
this separator seems long. it might wrap to the next line depending on the size of the window.